x-pack/plugins/endpoint/public/embeddables/resolver/view/process_event_dot.tsx

```diff
@@ -235,31 +257,11 @@ export const ProcessEventDot = styled(
                   ref={animationTarget}
                 />
               </use>
-              <use
-                role="presentation"
-                xlinkHref={`#${SymbolIds.processNodeLabel}`}
-                x={markerPositionOffset + markerSize - 0.5}
-                y={labelYOffset}
-                width={(markerSize / 1.7647) * 5}
-                height={markerSize / 1.7647}
-                opacity="1"
-                fill={labelFill}
-              />
-              <text
-                x={markerPositionOffset + 0.7 * markerSize + 50 / 2}
-                y={labelYOffset + labelYHeight / 2}
-                textAnchor="middle"
-                dominantBaseline="middle"
-                fontSize="3.75"
-                fontWeight="bold"
-                fill={NamedColors.empty}
-                paintOrder="stroke"
-                tabIndex={-1}
-                style={{ letterSpacing: '-0.02px' }}
-                id={labelId}
-              >
-                {eventModel.eventName(event)}
-              </text>
+              <foreignObject x="7.41" y="-3.4" height="6.4px" width="42px">
```

Comment:
Won't having arbitrary width and height here (and maybe elsewhere in the SVG) cause the `span` below to be scaled? It would be nice if the span was the expected size (the size you'd expect it to be outside of the SVG).

Reply:
I don't understand your question: everything in SVG is scaled (the _S_ in SVG) and positioned with coordinates.